`esy-skia` fails to build on Manjaro

`esy skia` requires two additional manual dependencies, `nasm` and `clang`. Manjaro packages clang 9, which unfortunately fails to build `skia` with the following error:

```
    ../../src/gpu/ops/GrOp.h:148:37: error: empty expression statement has no effect; remove unnecessary ';' to silence this warning [-Werror,-Wextra-semi-stmt]
            TRACE_EVENT0("skia", name());
```

This is because `-Werror` is on and clang 8+ includes a new warning, `extra-semi-stmt`.

I've tried installing clang 7, but that's installed side-by-side and unfortunately esy does not seem to respect the `CC` and `CXX` environment variables.

--- a/gn/BUILD.gn
+++ b/gn/BUILD.gn
@@ -364,6 +364,7 @@ config("warnings") {
 
       "-Wno-deprecated-declarations",
       "-Wno-maybe-uninitialized",
+      "-Wno-extra-semi-stmt",
     ]
     cflags_cc += [
       "-Wnon-virtual-dtor",

